packages/commonwealth/client/scripts/views/components/edit_profile.tsx

```diff
@@ -361,36 +354,10 @@ export default class EditProfileComponent extends ClassComponent<EditNewProfileA
               </div>
             </CWFormSection>
             <CWFormSection
-              title="Personalize Your Profile"
-              description="Express yourself through imagery."
+              title="Personalize your profile"
+              description="Use imagery to express yourself."
             >
-              <CWText fontWeight="medium">Cover Image</CWText>
```

Comment:
We should also remove the cover image code from the view profile page! as well as from the migration/data model!

Reply:
Good catch!